packages/editor/__mocks__/nanoid.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
Could you maybe patch the nanoid package instead? That seems a bit of an ugly hack to have to add this to every package that indirectly depends on nanoid. We should generally try to get our dependencies to build properly and be compatible with our own packages.